doc/21-development.md

```diff
@@ -1883,7 +1882,6 @@ Icinga needs the development header and library files from the Boost library.
 
 Visual Studio translates into the following compiler versions:
 
-- `msvc-14.1` = Visual Studio 2017
```

Comment:
Kinda strange: You refer with plural to a list of items... but there's only one item.

Reply:
Do you just want me to change "versions" into "version" or do you want a complete rephrasing? I didn't want to change too much as a newer version of Visual Studio will probably exist at some point, so any larger change would eventually have to be reverted.

Reply:
Fine.